__tests__/app.test.js

```diff
@@ -290,3 +286,68 @@ test("should return the array of objects in descing order", () => {
 
 })
 })})})
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should an array of comments for the given article_id and contain correct properties", () => {
```

Comment:
It should have another test to check if we can get a 200 when the ID is valid, but it has no comments (should return an empty array)